.storybook/pages/AlongDemo/AlongDemo.module.css

```diff
@@ -200,7 +223,6 @@
   margin-left: 1.5rem;
   display: flex;
   flex-direction: column;
-  width: 24.125rem;
```

Comment:
Previously the reflection text and response text area were going off the screen on mobile.